tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -4235,14 +4245,14 @@ This is a paragraph.
 * Blue
 
 \`\`\`markdown
-* Red
-* Green
-* Blue
-
 - Red
 - Green
 - Blue
 
++ Red
```

Comment:
These changes are caused by `node.tokens.length === 0` since there's no `tokens` in a `root` so it threw, then our multi-parser caught the error and decided not to format these codeblocks.

And sorry for the `TypeError: Cannot read property 'type' of undefined` error, I forgot to check `parent.children.length !== 0` before accessing `parent.children[0]`.

https://github.com/prettier/prettier/blob/8015e417c01a2417f9da8c15c2540258716378e7/src/language-markdown/printer-markdown.js#L805-L811

Reply:
@ikatyang got it

Reply:
@ikatyang So I'm gonna wait for the fix of `parent.children.length` or just create another PR for that?

Reply:
We can fix it here since we found it in this PR.